src/main/java/org/zalando/planb/revocation/api/ExceptionsResource.java

```diff
@@ -25,6 +26,14 @@
 @Slf4j
 public class ExceptionsResource {
 
+    @ExceptionHandler(RevocationUnauthorizedException.class)
+    @ResponseStatus(HttpStatus.UNAUTHORIZED)
```

Comment:
After reading the spec, I changed my mind :) - See [W3 specs](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html).